msg/tools/px_generate_uorb_topic_files.py

```diff
@@ -43,6 +43,7 @@
 import filecmp
 import argparse
 import sys
+import numpy as np
```

Comment:
Why did you add this?

Reply:
Oh yeah sorry. This was when I was comparing types. Will remove.